# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../noteedit/uic.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur noteedit-2.8.0.orig/noteedit/staffProps.ui noteedit-2.8.0/noteedit/staffProps.ui
--- noteedit-2.8.0.orig/noteedit/staffProps.ui	2005-07-05 21:46:39.000000000 +0300
+++ noteedit-2.8.0/noteedit/staffProps.ui	2005-10-22 18:12:02.000000000 +0300
@@ -543,13 +543,5 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
 </includehints>
 </UI>
diff -Nur noteedit-2.8.0.orig/noteedit/timesigDia.ui noteedit-2.8.0/noteedit/timesigDia.ui
--- noteedit-2.8.0.orig/noteedit/timesigDia.ui	2005-07-05 21:46:39.000000000 +0300
+++ noteedit-2.8.0/noteedit/timesigDia.ui	2005-10-22 18:14:56.000000000 +0300
@@ -357,7 +357,5 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
 </includehints>
 </UI>
